npm/qsharp/ux/circuit-vis/metadata.ts

```diff
@@ -15,7 +15,7 @@ export enum GateType {
   Swap,
   /** X gate. */
   X,
-  /** |0〉 or |1〉 gate. */
+  /** |0⟩or |1⟩ gate. */
```

Comment:
Another place to check characters.

Reply:
See from line 330 in util.ts. I checked all the files and made them consistent.